docs/develop/automatically-open-on-installation.md

```diff
@@ -0,0 +1,46 @@
+
+> [!NOTE]
+> 
+> - This feature is currently available only in Office on the web. We are working to bring this behavior to other platforms, but currently they still exhibit the historic default behavior described earlier.
+> - This feature applies only to add-ins installed by an end-user, not to centrally deployed add-ins.
+> - This feature doesn't apply to Content add-ins or Mail (Outlook) add-ins.
+> - This feature applies only to add-ins that have at least one add-in command of [the type "task pane command"](../design/add-in-commands.md#types-of-add-in-commands).
+
+## New behavior
+
+The new behavior is as follows:
+
+- If the add-in has just one [task pane command](../design/add-in-commands.md#types-of-add-in-commands), then the add-in's ribbon tab is selected and the task pane opens automatically upon installation. You don't need to configure anything.
+- If the add-in has multiple task pane commands, and one is configured to be the default (see [Configure default task pane](#configure-default-task-pane)), then the add-in's ribbon tab is selected and the default task pane opens automatically upon installation.
+- If the add-in has multiple task pane commands, but none is configured to be the default, then the add-in's ribbon tab is selected automatically upon installation and a callout appears near it notifying the user of the new add-in, but no task pane is opened. This is the same as the historic default behavior.
+
+> [!NOTE]
+> If for any reason, the add-in command that launches the task pane cannot be manually selected by a user at start up, such as when it's [configured to be disabled](../design/disable-add-in-commands.md) at start up, then it won't be automatically opened regardless of configuration. 
+
+## Configure default task pane
+
+To designate a task pane as the default, add a [TaskpaneId](/javascript/api/manifest/action#taskpaneid) element as the first child of the **\<Action\>** element and set its value to **Office.AutoShowTaskpaneWithDocument**. The following is an example.
+
+```xml
+<Action xsi:type="ShowTaskpane">
+    <TaskpaneId>Office.AutoShowTaskpaneWithDocument</TaskpaneId>
+    <SourceLocation resid="Contoso.Taskpane.Url" />
+</Action>
+```
+
+> [!TIP]
+> If you want your add-in to automatically launch whenever the user reopens the document, you need to take further configuration steps. For details and advice about when to use this feature, see [Automatically open a task pane with a document](automatically-open-a-task-pane-with-a-document.md). 
+
```

Comment:
Usually articles conclude with a See also section. What is the reasoning for not including a See also in this article?